website/src/pages/ProjectCard/card.css

```diff
@@ -1,52 +1,74 @@
 .card {
-  width: 500px;
-  height: 250px;
-  margin-bottom: 10%;
-  margin-left: 15%;
+  /* width: 500px;
```

Comment:
we have classes for this in tailwind CSS. Please use that. Also remove the commented out code